crates/bytecode/src/eof/verification.rs

```diff
@@ -558,9 +558,7 @@ pub fn validate_eof_code(
                 // Mark called code as accessed.
                 tracker.access_code(section_i);
 
-                // We decrement by `types.inputs` as they are considered as send
-                // to the called code and included in types.max_stack_size.
-                if this_instruction.biggest - stack_requirement + target_types.max_stack_size as i32
+                if this_instruction.biggest + target_types.max_stack_increase as i32
```

Comment:
We can now remove `- target_type.inputs` as max stack size became max_stack_increase, it is easier to calculate

There are few more places where this is done